### DISCUSSION TITLE
Trying to set up a domain name bought off of Google Domains

### DISCUSSION THREAD

**Discussion by TomerKrayzman** [community/community#21943] [Other Feature Feedback, Questions, & Ideas]
<p>Hi all!</p>
<p>I’m trying to set up a custom domain I bought off Google domains (<a href="http://tomerkrayzman.com" rel="nofollow noopener">tomerkrayzman.com</a>), and I set it up on Google domains that the page redirects to my personal site hosted on pages at <a href="http://tomerkrayzman.github.io" rel="nofollow noopener">tomerkrayzman.github.io</a>. </p>
<p>After setting my custom domain name to <a href="http://tomerkrayzman.com" rel="nofollow noopener">tomerkrayzman.com</a> in my settings, the website doesn’t seem to be working at all and just says that I’ve been rerouted too many times. The website starts working again when I take the custom domain name out of my settings (which also removes the CNAME file from the website’s repository). </p>
<p>Could this be happening because I only have a master branch and no gh-pages branch in my domain? Here’s a link to my repository: <a href="https://github.com/TomerKrayzman/TomerKrayzman.github.io" rel="nofollow noopener">https://github.com/TomerKrayzman/TomerKrayzman.github.io </a></p>
<p>I also attached what I’ve set up so far in Google Domains - not sure if I did everything correctly here either.</p>
<p>Any help is appreciated!</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/2278e577385d3721217e378abb43b258c878ed8b.png" data-download-href="/uploads/short-url/4UXfHq8EV54pUh7U7HmZiqR4rG3.png?dl=1" title="sc.PNG" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/11336168/181067606-895dffea-7bc0-4549-b556-14be43f6fa9b.png" alt="sc.PNG" data-base62-sha1="4UXfHq8EV54pUh7U7HmZiqR4rG3" width="690" height="404" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/2278e577385d3721217e378abb43b258c878ed8b_2_690x404.png, https://aws1.discourse-cdn.com/github/optimized/1X/2278e577385d3721217e378abb43b258c878ed8b_2_1035x606.png 1.5x, https://aws1.discourse-cdn.com/github/original/1X/2278e577385d3721217e378abb43b258c878ed8b.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/2278e577385d3721217e378abb43b258c878ed8b_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">sc.PNG</span><span class="informations">1227×720 30.4 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by sin9yt** [comment, marked as answer]
<p>Hello Tromer,</p>
<p>From what I see, you are creating a loop of redirects. </p>
<p>To solve this issue do the following:</p>
<ul>
<li>
<p>Under the <strong>Domain Forward</strong> section of your DNS settings, click Edit</p>
</li>
<li>
<p>Change it from <strong><a href="http://tomerkrayzman.github.io" rel="nofollow noopener">tomerkrayzman.github.io</a></strong> to <strong><a href="http://host.tomerkrayzman.com" rel="nofollow noopener">host.tomerkrayzman.com</a></strong></p>
</li>
</ul>
<p>This will solve it. If you still have issues, let me know.</p>
<p>Regards,</p>
<p>sin9yt</p>
